### DISCUSSION TITLE
Can I switch from create-react-app to create-next-app automatically using a CLI?

### DISCUSSION THREAD

**Discussion by romw314** [vercel/next.js#57868] [Help] [answered]
### Summary

Title describes the question. I want something like `npx create-next-app --migrate`.

**Comment by icyJoseph** [comment, marked as answer]
Something like this? https://nextjs.org/docs/app/building-your-application/upgrading/codemods#migrate-from-cra 

Never used it myself though.

**Comment by romw314** [comment]
Thanks. Some CLI tools aren't good for me, but I can always `git reset --hard`.
